formfyxer/lit_explorer.py

```diff
@@ -794,6 +798,66 @@ def needs_calculations(text: Union[str, Doc]) -> bool:
     return False
 
 
+def highlight_text(sentence:str, fragments:str, highlight_class:str="highlight")->str:
+    highlighted = sentence
+    for frag in fragments:
+        highlighted = re.sub(frag, f'<span class="{ highlight_class }">{frag}</span>', sentence)
+    return highlighted
```

Comment:
This belongs in PDFStats, not in here. Should factor it out. There are a lot of different ways we might report back this data, and PDFStats is the HTML package.

Reply:
Fair enough. Was having trouble with passing a better data structure and serializing it, but I can do just a dictionary or something similar.